src/_nested-list.scss

```diff
@@ -81,6 +87,30 @@ $button: #{$fd-namespace}-button;
         border-top: none;
       }
     }
+
+    &--popover {
+      &:first-child {
+        border-top-left-radius: $border-radius;
+        border-top-right-radius: $border-radius;
+
+        .#{$block}__link,
+        .#{$block}__content {
+          border-top-left-radius: $border-radius;
+          border-top-right-radius: $border-radius;
+        }
+      }
+
+      &:last-child {
+        border-bottom-left-radius: $border-radius;
+        border-bottom-right-radius: $border-radius;
+
+        .#{$block}__link,
+        .#{$block}__content {
+          border-bottom-left-radius: $border-radius;
+          border-bottom-right-radius: $border-radius;
```

Comment:
repeatable code, should be moved to a mixin or placeholder

Reply:
created and moved to mixing.